test/commands/scanner/rule/remove.test.ts

```diff
@@ -48,9 +48,14 @@ describe('scanner:rule:remove', () => {
 		describe('Rule Removal', () => {
 			describe('Test Case: Removing a single PMD JAR', () => {
 				removeTest
-					// We'll wait three seconds then send in a 'y', to simulate the user confirming the request.
-					.stdin('y\n', 3000)
-					.timeout(10000)
+					// We'll wait ten seconds then send in a 'y', to simulate the user confirming the request.
+					// Note: The real time until the prompt is given shouldn't be anywhere near this long,
+					// but a high delay prevents test failures due to the input being sent before the prompt
+					// is displayed.
+					.stdin('y\n', 10000)
```

Comment:
Can we reduce the stdin time to 5 seconds? Though I still like the idea of increasing the test's timeout to 20 seconds just to be sure.

Reply:
The stdin time being 5 seconds was what caused the problem.
For whatever reason, we've added enough stuff to the beginning of the transaction that it now sometimes takes longer than 5 seconds for the prompt to display.